packages/expansion-hub/src/errors/ParameterOutOfRangeError.ts

```diff
@@ -0,0 +1,11 @@
+import {NackError, setPrototypeOf} from "./NackError.js";
+
+export class ParameterOutOfRangeError extends NackError {
+    parameterIndex: number;
+    constructor(parameterIndex: number) {
+        super(parameterIndex, `Parameter ${parameterIndex} is out of range`);
```

Comment:
We should probably specify that the parameter number is zero-indexed (it is, right?)

Reply:
It is zero-indexed

Reply:
Documented this